lambdarest/__init__.py

```diff
@@ -97,7 +97,15 @@ def inner_lambda_handler(event, context=None):
 
         # Save context within event for easy access
         event["context"] = context
-        path = event['resource']
+
+        # check if resource and path are the same. If so, use the requested
+        # resource path, which will contain the actual requested path itself.
+        # If they are not the same, this is a proxied or custom domain where
+        # we need to use the event resource
+        if 'path' in event and event['resource'][0:3] == event['path'][0:3]:
```

Comment:
why do you only check the first 3 letters ?

Reply:
hmmm, no particular reason. I'll change it, and change it for the first part of the url, that's more accurate.